docs/api/components/schemas.yaml

```
@@ -364,29 +366,20 @@ Role:
         Role names can only have alphanumeric ASCII characters and underscores.
       type: string
     admin_channels:
-      description: The channels that users in the role are able to access for the default collection.
+      description: The channels that users in the role are able to access for the default collection. Use `collection_access.scopename.collectionname.admin_channels` for channels in named collections.
       type: array
       items:
         type: string
     all_channels:
       description: |-
-        The channels that the role grants access to for the default collection.
+        The channels that the role grants access to for the default collection. See `collection_access.scopename.collectionname` for channels in named collections.
```

New line for each sentence

```suggestion
        The channels that the role grants access to for the default collection.
        See `collection_access.scopename.collectionname` for channels in named collections.
```